test/library/packages/LinearAlgebra/performance/me/me-perf.chpl

```diff
@@ -0,0 +1,295 @@
+use Time;
+use Math;
+use LinearAlgebra;
+
+config var m=4,
+             iters=10,
+             /* Omit timing output */
+             correctness=false;
+
+config type eltType = real;
+
+const nbytes = numBytes(eltType);
+
+
+proc test_expm() {
+  const Space = {0..<m, 0..<m};
+  var t: Timer;
+
+  if !correctness {
+    writeln('=========================================');
+    writeln('  Matrix Exponential Performance Test');
+    writeln('=========================================');
+    writeln('iters : ', iters);
+    writeln('m     : ', m);
+    writeln('numLocales     : ', numLocales);
+    writeln('MB    : ', (nbytes*m*m) / 10**6);
+    writeln();
+  }
+
+  {
+    // Pade3, Pade5, Pade7, Pade9, Pade13 cases
+
+    var b = [0.01, 0.1, 0.5, 1.0, 10.0];
+    const pade = [3, 5, 7, 9, 13];
+
+    for i in 0..4 do {
```

Comment:
I think we want this to iterate over `b.domain`. Same applies to other similar loops in this file

Reply:
But as a more fundamental question: do we get huge performance differences with different values that we iterate over here?

Reply:
For Pade values, yes, there was a difference of 3-4 milli seconds for each case for a matrix of size 20